server/services/reporting.js

```diff
@@ -291,11 +291,53 @@ const getNoOfferParticipantsReport = async () => {
   }));
 };
 
+const getRosParticipantsReport = async () => {
+  const searchOptions = {};
+  const rosEntries = await dbClient.db[collections.ROS_STATUS]
+    .join({
+      participantJoin: {
+        type: 'LEFT OUTER',
+        relation: collections.PARTICIPANTS,
+        on: {
+          id: 'participant_id',
+        },
+      },
+      statusJoin: {
```

Comment:
Do we need to join status? I don't see any use of status data in the criteria or results.

Reply:
There's a need to include site and health region to csv export
Please, see below:
```
site: entry.siteJoin?.body?.siteName,
    healthRegion: entry.siteJoin?.body?.healthAuthority,
```

Reply:
My question is about `statusJoin`?